blockchain/full_client.py

```diff
@@ -469,12 +469,12 @@ def _handle_transaction(self, transaction, broadcast=False, print_nodes=False):
     def _broadcast_new_transaction(self, transaction, print_nodes=False):
```

Comment:
please renme this method too

Reply:
but this method actually broadcasts the transaction to all the neighbors?

Reply:
mh ok. Makes sense somehow